installation/index.qmd

```diff
@@ -73,10 +77,14 @@ To get the most out of Viash, you'll also need to install [Docker](https://docs.
 
 ### Installing Viash
 
-To install Viash to a `bin` folder in your current directory, run the following command to download and run the install script:
+To install Viash to a `bin` folder in your PATH, run the following command to download and run the install script:
 
 ```bash
-curl -fsSL dl.viash.io | bash; mv viash $HOME/bin
+wget -qO- dl.viash.io | bash
```

Comment:
Wasn't the reason the original command used `curl` that macOS doesn't necessarily have `wget` but was sure to have `curl`?